src/gui/clips/ClipView.cpp

```diff
@@ -998,20 +925,11 @@ void ClipView::mouseMoveEvent( QMouseEvent * me )
  */
 void ClipView::mouseReleaseEvent( QMouseEvent * me )
 {
-	// If the CopySelection was chosen as the action due to mouse movement,
-	// it will have been cleared.  At this point Toggle is the desired action.
-	// An active StringPairDrag will prevent this method from being called,
-	// so a real CopySelection would not have occurred.
-	if( m_action == CopySelection ||
-	    ( m_action == ToggleSelected && mouseMovedDistance( me, 2 ) == false ) )
```

Comment:
Because `mouseMoveEvent` clears `m_action` if the mouse moves. I'll clarify this in the comment like so:

> If we get here it means the user simply clicked without dragging, because that would
have been caught in mouseMoveEvent and m_action would be NoAction as a result

Reply:
You mean, if `mouseMovedDistance( me, 2 )` would be true, `mouseMoveEvent` would have been called?

Reply:
Yes exactly, you can't move the mouse without triggering a `mouseMoveEvent`. This is the case

```py
def mouseMoveEvent():
    if action == CopyOrToggle and mouseMovedDistance(2):
        doCopy()
        action = NoAction

def mouseReleaseEvent():
    if action == CopyOrToggle:
        doToggle()